ci/ansible/roles/pulp/tasks/main.yaml

```diff
@@ -137,7 +142,7 @@
       name: Pulp Project repository
       baseurl: "https://repos.fedorapeople.org/pulp/pulp/testing/automation/{{ pulp_version }}/stage/{% if ansible_distribution == 'Fedora' %}fedora-{% endif %}$releasever/$basearch/"
       gpgcheck: 0
-      priority: "{{ repo_priority }}"
+      priority: "{{ '50' if fips_state.stdout == '1' else repo_priority }}"
```

Comment:
@bherrin3, could we evaluate this just once - probably together with the fips verification, and pass the variable over here?

Reply:
@kersommoura yup. I didn't know a better way to do it at the time and I didn't like a static value sitting in there for fips.

Updating with the following logic:
- adding defaults for fips_priority, default_priority
- Using `set_fact` after fips eval to dynamically assign the above default to `repo_priority` variable
- Updating the priority statement to be `"{{ repo_priority }}"` above.

Thanks, @rochacbruno ! Re-testing now.